src/Microsoft.IdentityModel.Tokens/SymmetricSignatureProvider.cs

```diff
@@ -205,7 +205,7 @@ public override bool Verify(byte[] input, byte[] signature)
                 throw LogHelper.LogException<ObjectDisposedException>(typeof(SymmetricSignatureProvider).ToString());
 
             if (_keyedHash == null)
-                throw LogHelper.LogException<ArgumentException>(LogMessages.IDX10623);
+                throw LogHelper.LogException<ArgumentNullException>(LogMessages.IDX10623);
```

Comment:
This probably shouldn't be an ArgumentNullException because there isn't a null method argument. InvalidOperationException?